src/cljs/main/broadfcui/page/workspace/analysis/track_selector.cljs

```diff
@@ -17,6 +17,13 @@
 
 
 (def ^:private supported-file-types [".bam" ".vcf" ".bed"])
+;; IGV.js supports:
```

Comment:
Why does this list differ from the above line and the `case file-extension` block from igv.cljs?

Reply:
This PR maintains the list of supported file types - it neither adds nor removes any. Way back when, we limited the supported file types to reduce scope of testing and troubleshooting. This comment is my - I guess passive-aggressive - way of reminding ourselves that we could improve on this. However, any change to supported file types would require code changes in `find-index-file` and potentially elsewhere - as well as testing - so I'd rather not do it as part of this PR.

Reply:
Makes sense.  Then I would like a small comment update.  Something like "FireCloud currently supports the above subset of IGV file types.  The full list is..."

Reply:
👍to update